.github/pull_request_template.md

```diff
@@ -6,22 +6,27 @@ change me!
 
 ### User-facing documentation
 
-(*must be* 2 items and both *must be* checked)
-<!-- Remove conflicting items that won't be checked. -->
+(*must be **1** checked* item)
+<!-- remove two other items that don't apply -->
 
-- [ ] CHANGELOG is updated
-- [ ] CHANGELOG update is not needed
-- [ ] Documentation PR is created and linked above
-- [ ] Documentation is not needed
+- [ ] I added Release Note(s) to CHANGELOG.md
+- [ ] I [requested](/dev/null) Release Notes update on the JIRA ticket: <!-- put JIRA ticket link here -->
+- [ ] No Release Notes update is needed
```

Comment:
I think it would be good to have a format that helps with the kind of checks prepared in #11716
Would it make sense to have something like:
```
- [ ] Release Notes added to CHANGELOG
- [ ] Release Notes requested through JIRA ticket <!-- put JIRA ticket link here -->
- [ ] Release Notes not needed
```
and
```
- [ ] Documentation PR [created](...): <!-- put the PR link here -->
- [ ] Documentation requested from Documentation team: <!-- put JIRA ticket or Slack link here -->
- [ ] Documentation - No user-facing Documentation is needed
```

Reply:
This looks good from my end.

Reply:
Lgtm, might say “must check one” for brevity. TY for doing this!